packages/core/src/common/message-rpc/rpc-protocol.ts

```diff
@@ -191,8 +189,8 @@ export class RpcProtocol {
     }
 
     cancelError(): Error {
-        const error = new Error('"Request has already been canceled by the sender"');
-        error.name = 'Cancel';
+        const error = new Error('Canceled');
```

Comment:
If we rely on these values to be the same as in VS Code, we really should have a comment explaining the fact and pointing to the VS Code code. Otherwise someone _will_ "fix" the message at some point.

Reply:
Actually I just discovered that a corresponding `CancellationError` is already available (introduced with https://github.com/eclipse-theia/theia/pull/10035) so I removed the custom error handling and we throw a new `CancellationError` instead.